lib/newrelic-slack-ruby-bot/instrumentation.rb

```diff
@@ -30,4 +30,21 @@ def message_with_new_relic(client, data)
       alias_method :call, :message_with_new_relic
     end
   end
+
+  executes do
+    ObjectSpace.each_object(::SlackRubyBot::Commands::Base.singleton_class) do |command_class|
+      command_class.class_eval do
+        class << self
+          def call_command_with_new_relic(client, data, match, block)
+            ::NewRelic::Agent.set_transaction_name("#{self}/#{match[:command].downcase}")
```

Comment:
Why downcase? Because the matching is not case-sensitive?

Reply:
New Relic separates transactions by name and is case sensitive. So if someone types `@mybot Command` it would show up separately from `@mybot command`. Downcasing normalizes the names so that individual commands are aggregated in New Relic.